raiden_contracts/deploy/__main__.py

```diff
@@ -504,9 +504,15 @@ def verify_deployed_contracts(web3: Web3, contract_manager: ContractManager, dep
     receipt = web3.eth.getTransactionReceipt(
         contracts[CONTRACT_ENDPOINT_REGISTRY]['transaction_hash'],
     )
-    assert receipt['blockNumber'] == contracts[CONTRACT_ENDPOINT_REGISTRY]['block_number']
-    assert receipt['gasUsed'] == contracts[CONTRACT_ENDPOINT_REGISTRY]['gas_cost']
-    assert receipt['contractAddress'] == contracts[CONTRACT_ENDPOINT_REGISTRY]['address']
+    assert receipt['blockNumber'] == contracts[CONTRACT_ENDPOINT_REGISTRY]['block_number'], \
```

Comment:
Just an FYI for the future -- no need to change this here: 

In raiden we do multiline asserts like this:

```python
assert a == b , (
    "A very big"
    "string here"
)
```